EventGenerator/src/CaloCalibGun_module.cc

```diff
@@ -0,0 +1,252 @@
+// Simulate the photons coming from the pipe calibration source
+// based on CaloCalibGun orginally written by Bertrand Echenard (2014)
+// Current module author: Sophie Middleton (2022)
+
+// C++ includes.
+#include <iostream>
+#include <string>
+#include <cmath>
+
+// Framework includes
+#include "art/Framework/Principal/Run.h"
+#include "art_root_io/TFileDirectory.h"
+#include "art_root_io/TFileService.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Services/Registry/ServiceHandle.h"
+
+// Mu2e includes
+#include "Offline/GeometryService/inc/GeomHandle.hh"
+#include "Offline/DataProducts/inc/PDGCode.hh"
+#include "Offline/SeedService/inc/SeedService.hh"
+#include "Offline/MCDataProducts/inc/ProcessCode.hh"
+#include "Offline/CalorimeterGeom/inc/DiskCalorimeter.hh"
+#include "Offline/Mu2eUtilities/inc/RandomUnitSphere.hh"
+#include "Offline/MCDataProducts/inc/GenId.hh"
+#include "Offline/MCDataProducts/inc/GenParticle.hh"
+#include "Offline/MCDataProducts/inc/PrimaryParticle.hh"
+#include "Offline/MCDataProducts/inc/MCTrajectoryCollection.hh" 
+
+// Other external includes.
+#include "CLHEP/Random/RandFlat.h"
+#include "CLHEP/Vector/ThreeVector.h"
+#include "CLHEP/Vector/LorentzVector.h"
+#include "CLHEP/Random/RandomEngine.h"
+#include "CLHEP/Random/RandExponential.h"
+#include "CLHEP/Random/RandPoissonQ.h"
+
+
+#include "fhiclcpp/types/Atom.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+
+//ROOT Includes
+#include <TMath.h>
+
+using CLHEP::degree;
+using namespace std;
+using namespace mu2e;
+
```

Comment:
You no longer use TMath - remover the header.  You always spell out CLHEP::degree in full - I like that.  So remove the using CLHEP::degree.  And you do not need using namespace mu2e - remove it.